Doc/library/pathlib.rst

```diff
@@ -21,6 +21,15 @@ inherit from pure paths but also provide I/O operations.
 .. image:: pathlib-inheritance.png
    :align: center
    :class: invert-in-dark-mode
+   :alt: Inheritance diagram showing the classes available in pathlib. The
+         most basic class is PurePath, which has three direct subclasses:
+         PurePosixPath, PureWindowsPath, and Path. Further to these four
+         classes, there are two classes that use multiple inheritance:
+         PosixPath subclasses PurePosixPath and Path, and WindowsPath
+         subclasses PureWindowsPath and Path. The upper part of the diagram
+         shows the three pure classes, starting with PurePath, and the lower
+         part shows the three concrete classes, ending with PosixPath and
+         WindowsPath.
```

Comment:
I think we can skip this bit, we've already described the important parts of the diagram, we don't need to describe visual layout.

```suggestion
         subclasses PureWindowsPath and Path.
```